src/platforms/rcore_desktop_sdl.c

```diff
@@ -994,17 +1011,6 @@ void PollInputEvents(void)
     // Register previous touch states
     for (int i = 0; i < MAX_TOUCH_POINTS; i++) CORE.Input.Touch.previousTouchState[i] = CORE.Input.Touch.currentTouchState[i];
 
-    // Reset touch positions
-    // TODO: It resets on target platform the mouse position and not filled again until a move-event,
-    // so, if mouse is not moved it returns a (0, 0) position... this behaviour should be reviewed!
-    //for (int i = 0; i < MAX_TOUCH_POINTS; i++) CORE.Input.Touch.position[i] = (Vector2){ 0, 0 };
-
-    // Map touch position to mouse position for convenience
-    // WARNING: If the target desktop device supports touch screen, this behavious should be reviewed!
-    // https://www.codeproject.com/Articles/668404/Programming-for-Multi-Touch
-    // https://docs.microsoft.com/en-us/windows/win32/wintouch/getting-started-with-multi-touch-messages
```

Comment:
This is done by default in SDL_touch.c anyhow:

SDL_touch.c:34
```c
#define SYNTHESIZE_TOUCH_TO_MOUSE 1
```

Reply:
So, it has been restored:

```c
        CORE.Input.Touch.position[0] = CORE.Input.Mouse.currentPosition;
```

I deleted the comments:

>     // WARNING: If the target desktop device supports touch screen, this behavious should be reviewed!

It is hereby reviewed!

>    // https://www.codeproject.com/Articles/668404/Programming-for-Multi-Touch
>    // https://docs.microsoft.com/en-us/windows/win32/wintouch/getting-started-with-multi-touch-messages

These are hangovers from rcore_desktop.c being used as a template for this backend.

GLFW doesn't support touch, so Ray linked to docs for Windows touch as a future reference.

But not relevant here as SDL implemented that :-)